code/src/index.css

```diff
@@ -5,9 +5,143 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-}
+  background-color: black;
+  }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: Arial, Helvetica, sans-serif;
+ /*  source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace; */
+    display: flex;
+}
+
+
+.dots, .heart, .play {
+  height: 60px;
+  width: 46px;
+  filter: invert(100%) sepia(0%) saturate(7493%) hue-rotate(299deg) brightness(103%) contrast(105%);
```

Comment:
We have approached the style of icons by writing the codes like this: 


 ` .image-wrapper:hover .icons {
    display: flex;
    opacity: 1;
  }

  .image-wrapper:hover .album-cover {
    opacity: 0.3;
  }

  .album-cover {
    transition: .5s ease;
  }

  .icons {
    transition: .5s ease;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    -ms-transform: translate(-50%, -50%);
  }

  /* HOVER ON PLAY BUTTON */

  .grow {
    transition: all .2s ease-in-out;
  }

  .grow:hover {
    transform: scale(1.5);
  }
`